pkg/server/container_create.go

```diff
@@ -213,7 +216,13 @@ func (c *criContainerdService) generateContainerSpec(id string, sandboxPid uint3
 
 		// TODO(random-liu): [P1] Set selinux options.
 
-		// TODO(random-liu): [P2] Add apparmor and seccomp.
+		// TODO(random-liu): [P2] Add seccomp.
+
+		// Set apparmor profile name, Note: trims default profile name prefix
+		appArmorProf := securityContext.GetApparmorProfile()
```

Comment:
For `runtime/default`, we should have a default profile. For now, it could be empty.
But a TODO is needed here to make the default profile configurable from flag.